src/applications/ivc-champva/10-7959C/chapters/healthInsuranceInformation.js

```diff
@@ -51,7 +51,7 @@ export function applicantHasInsuranceSchema(isPrimary) {
       ...titleUI(
         ({ formData }) =>
           `${nameWording(formData, undefined, undefined, true)} ${
-            isPrimary ? 'primary' : 'secondary'
+            isPrimary ? '' : 'additional'
```

Comment:
We drop primary/secondary label and just use blank for primary insurance and 'additional' for secondary. This is done in a couple places.